apps/wallet-mobile/src/components/QRCodeScanner/QRCodeScanner.tsx

```diff
@@ -182,7 +182,7 @@ export const getScaledQrBounds = ({
   deviceHeight: number
   deviceWidth: number
 }) => {
-  // qr bounds values are inversely proportioned + issues in some android devices https://github.com/expo/expo/issues/17795
```

Comment:
that bug its fixed in latest versions. Also, the qr position data is not optional, as the comments in that issue suggest